s32/soc/s32z270/include/Hse_Ip_Cfg.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021-2024 NXP
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef HSE_IP_CFG_H
+#define HSE_IP_CFG_H
+
+/**
+*   @file
+*
+*   @addtogroup CRYPTO_43_HSE
+*   @{
+*/
+
+#ifdef __cplusplus
+extern "C"{
+#endif
+
+
+/*==================================================================================================
+*                                          INCLUDE FILES
+* 1) system and project includes
+* 2) needed interfaces from external units
+* 3) internal and external interfaces from this unit
+==================================================================================================*/
+#include "S32Z2_MU.h"
+#include "OsIf.h"
+
+/*==================================================================================================
+*                                 SOURCE FILE VERSION INFORMATION
+==================================================================================================*/
+#define HSE_IP_CFG_VENDOR_ID_H                      43
+#define HSE_IP_CFG_SW_MAJOR_VERSION_H               2
+#define HSE_IP_CFG_SW_MINOR_VERSION_H               0
+#define HSE_IP_CFG_SW_PATCH_VERSION_H               0
+
+/*==================================================================================================
+*                                       FILE VERSION CHECKS
+==================================================================================================*/
+
+/*==================================================================================================
+*                                            CONSTANTS
+==================================================================================================*/
+
+/*==================================================================================================
+*                                       DEFINES AND MACROS
+==================================================================================================*/
+
+/* Defines for the available MU instances */
+#define HSE_IP_MU_0                          ((uint8)0U)
+#define HSE_IP_MU_1                          ((uint8)1U)
+#define HSE_IP_MU_2                          ((uint8)2U)
+#define HSE_IP_MU_3                          ((uint8)3U)
+#define HSE_IP_MU_4                          ((uint8)4U)
+#define HSE_IP_MU_5                          ((uint8)5U)
+#define HSE_IP_MU_6                          ((uint8)6U)
+#define HSE_IP_MU_7                          ((uint8)7U)
+
+/* Max number of MU instances */
+#define HSE_IP_NUM_OF_MU_INSTANCES               (8)
```

Comment:
shouldn't this depend on the MU devicetree nodes enabled?

Reply:
It's the maximum number of MU that the chip supports, not the number of MU devicetree nodes enabled.